src/Components/Products/index.js

```diff
@@ -1,20 +1,21 @@
-import Card from "@mui/material/Card";
-import CardContent from "@mui/material/CardContent";
-import Typography from "@mui/material/Typography";
+import  { CardHeader }  from "@mui/material";
+import  Card  from "@mui/material/Card";
+import  CardContent  from "@mui/material/CardContent";
+import  Typography  from "@mui/material/Typography";
 import React from 'react';
 
-const Products = ({ product }) => {
+const Products = (product) => {
   return (
-    
-    <Card>
-      <CardContent>
-        <Typography mt={2} component="div">
-           {product.name}
+    <Card variant="outlined">
+     <CardHeader title={product.name} subheader={product.category}/>
```

Comment:
If the name is not showing try changing `product.name`  to `product.product.name` because it is inside of an object inside another object:
```
{
product:
  {
    product:{
      name:'TV'
      category:'electronics'
      price:699.00
      inStock:5
    }
  }
}
```